ui-react/packages/atlasmap/src/UI/ExpressionContent.tsx

```diff
@@ -245,7 +246,7 @@ export const ExpressionContent: FunctionComponent<IExpressionContentProps> = ({
    *
    * @param event - expression keyboard event
    */
-  function onKeyDown(event: any): void {
+  function onKeyDown(event: React.KeyboardEvent<HTMLDivElement>): void {
```

Comment:
Better import the elements explicitly than referring to them from the default export. So in this case `import React, { KeyboardEvent } from "react"` and then use it directly `KeyboardEvent<HTMLDivElement>`.

This helps addressing a number of issues, some with Storybook and some with the tree shaking performed by the consumer of this library.